src/cli/commands/start/start.ts

```diff
@@ -298,4 +306,20 @@ export async function start(options: SWACLIConfig) {
     .catch((err) => {
       logger.error(err.message, true);
     });
+
+  const end = new Date().getTime();
+
+  collectTelemetryEvent(
+    TELEMETRY_START_EVENT,
+    {
+      macAddressHash: (await getMachineId("sha256", TELEMETRY_MAC_ADDRESS_HASH_LENGTH)).toString(),
```

Comment:
Can these properties be added at the common layer so that we don't need to force every caller of `collectTelemetryEvent` to send this information if it is common to all the telemetry events